.github/workflows/build-linux-arm64-installer.yml

```diff
@@ -272,7 +272,7 @@ jobs:
           build_scripts/final_installer/chia-blockchain-cli_${CHIA_INSTALLER_VERSION}-1_arm64.deb
 
     - name: Mark installer complete
-      if: steps.check_secrets.outputs.HAS_GLUE_SECRET && env.FULL_RELEASE == 'true'
+      if: steps.check_secrets.outputs.HAS_GLUE_SECRET && env.RELEASE == 'true'
```

Comment:
Glue is configured with two different jobs: `chia-blockchain` and `chia-blockchain-prerelease`
We need to make one call for FULL_RELEASE and a separate call for prerelease, because the url is different.

For full release, its what you have here already. For the prerelease call, its:
`${{ secrets.GLUE_API_URL }}/api/v1/$RFC_REPO-prerelease/$RELEASE_TAG/success/build-arm`

The same pattern applies for all the prerelease calls - after `v1`, it needs to be `chia-blockchain-prerelease`

Reply:
Updated